contracts/EpochTokenLocker.sol

```diff
@@ -71,6 +78,11 @@ contract EpochTokenLocker {
             "withdraw was not registered previously"
         );
 
+        require(
+            !hasCreditedBalance[msg.sender][token][getCurrentBatchId()],
+            "withdraw is not possible, due to new credit in this batchId"
```

Comment:
Maybe more clear "Withdraw not possible for token that is traded in the current auction." ?